src/applications/personalization/profile360/util/index.js

```diff
@@ -59,26 +62,33 @@ export const hasInvalidWorkPhoneNumberError = errors =>
   hasErrorMessage(errors, GENERIC_ERROR_KEY, 'day phone number') ||
   hasErrorMessage(errors, GENERIC_ERROR_KEY, 'day area number');
 
+export const hasPaymentRestrictionIndicatorsError = errors =>
+  hasErrorMessage(errors, PAYMENT_RESTRICTIONS_PRESENT_KEY);
+
 // Helper that creates and returns an object to pass to the recordEvent()
 // function when an errors occurs while trying to save/update a user's direct
 // deposit payment information. The value of the `error-key` prop will change
 // depending on the content of the `errors` array.
-export const createDirectDepositAnalyticsDataObject = errors => {
+export const createDirectDepositAnalyticsDataObject = (errors = []) => {
```

Comment:
Did a minor refactor here. Setting a default value for the `errors` param so we don't need to make sure it exists before checking its length. Am also no longer editing the error object in the conditionals; instead just setting an error code in the conditionals and then building the object later.